scripts/darwin.sh

```diff
@@ -62,5 +66,23 @@ for disk in $DISKS; do
     echo "system: False"
   fi
 
+  partitions="`diskutil list $disk | get_device_identifiers`"
```

Comment:
Notice that the list of partitions that you get from `diskutil list <disk>` is a subset of the whole `diskutil list` output.

For example:

```
$ diskutil list
/dev/disk0 (internal, physical):
   #:                       TYPE NAME                    SIZE       IDENTIFIER
   0:      GUID_partition_scheme                        *251.0 GB   disk0
   1:                        EFI EFI                     209.7 MB   disk0s1
   2:          Apple_CoreStorage Macintosh HD            250.1 GB   disk0s2
   3:                 Apple_Boot Recovery HD             650.0 MB   disk0s3
/dev/disk1 (internal, virtual):
   #:                       TYPE NAME                    SIZE       IDENTIFIER
   0:                  Apple_HFS Macintosh HD           +249.8 GB   disk1
                                 Logical Volume on disk0s2
                                 DCD23031-6322-4269-A142-CD36C8FD95D7
                                 Unlocked Encrypted
/dev/disk2 (external, physical):
   #:                       TYPE NAME                    SIZE       IDENTIFIER
   0:     FDisk_partition_scheme                        *7.8 GB     disk2
   1:                 DOS_FAT_32 UNTITLED                7.8 GB     disk2s1
```
***
```
$ diskutil list /dev/disk2
/dev/disk2 (external, physical):
   #:                       TYPE NAME                    SIZE       IDENTIFIER
   0:     FDisk_partition_scheme                        *7.8 GB     disk2
   1:                 DOS_FAT_32 UNTITLED                7.8 GB     disk2s1
```

Notice that the `diskutil list /dev/disk2` output is just the bottom part of `diskutil list`. This means we could re-use the initial `diskutil list` call at line 15 somehow to also get the partitions to avoid an extra `diskutil` call.